File: C++/9.invertedtriangle.cpp
/*Create this pattern

* * * * * * 
 * * * * * 
  * * * * 
   * * * 
    * * 
     * 

*/

#include <iostream>
using namespace std;

int main() {
    int N;
    cout<<"Enter no. of rows: ";
    cin>>N;
    
    for (int i = 0; i<N; i++){
        for (int j = 0; j < i; j++){
            cout<<" ";
        }
        
        for (int j = 0; j <N-i; j++){
            cout<<"* ";
        }
        
        cout<<endl;
    }
    

    return 0;
}

File: Python/9.invertedtriangle.py
'''Create this pattern

* * * * * * 
 * * * * * 
  * * * * 
   * * * 
    * * 
     * 
'''

row = int(input("Enter the no. of rows: "))

for i in range(row):
    for j in range(i):
        print(" ", end='')
    for j in range(row-i):
        print("* ", end='')
    print("\n")